api/RTCRtpTransceiverInit.json

```diff
@@ -77,10 +77,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": "11"
```

Comment:
This (`direction`) goes back to https://github.com/WebKit/WebKit/commit/7dd1e36c8aa178d8b76db4783cf38c773c90c430 way before Safari 11, so this seems correct.